### ISSUE TITLE
NSF Exercises 1, 2 & 3 (and all exercises) - Reduce white space on the Setup page

### ISSUE THREAD

**Issue by ald32** [starteam/starcellbio_html#157]
There is a lot of blank vertical space above the samples table. Can you remove some of this space? I think it's accounting for when the 'in the lab' feature used to be there, but we would still like to be consistent with other pages in terms of spacing at the top of the page.
https://scbdev.odl.mit.edu/#view=experiment_setup&assignment_id=scb_ex2&experiment_id=scb_Experiment_1g0b00e19oq6im

**Comment by annagav** [comment]
There is also some instructions in the middle. Should I remove white space above and below the text?

**Comment by ald32** [comment]
Yes, I think that would be great. A little bit of space would be good, but you can definitely reduce it to 1/4 of what it is currently. Thanks!
